install-host/oval.sh

```diff
@@ -1,36 +1,39 @@
 #! /bin/sh -
 
 if [ $# -eq 0 ]; then
-	echo "specify [--redhat --amazon --debian --ubuntu --alpine --oracle]"
+	echo "specify [--redhat --amazon --debian --ubuntu --alpine --oracle --fedora]"
 	exit 1
 fi
 
 target=$1
 shift
 
 case "$target" in
-	--redhat) 
-		goval-dictionary fetch redhat ${@} 6 7 8 
+	--redhat)
+		goval-dictionary fetch redhat ${@} 6 7 8
 		;;
-	--amazon) 
-		goval-dictionary fetch amazon ${@} 
+	--amazon)
+		goval-dictionary fetch amazon ${@}
 		;;
-	--debian) 
+	--debian)
 		goval-dictionary fetch debian ${@} 8 9 10 11
 		;;
-	--ubuntu) 
+	--ubuntu)
 		goval-dictionary fetch ubuntu ${@} 16 18 20
 		;;
-	--alpine) 
+	--alpine)
 		goval-dictionary fetch alpine ${@} 3.2 3.3 3.4 3.5 3.6 3.7 3.8 3.9 3.10 3.11 3.12 3.13 3.14
 		;;
 	--oracle)
 		goval-dictionary fetch oracle ${@}
 		;;
-	--*)  echo "specify [--redhat --amazon --debian --ubuntu --alpine --oracle]"
+	--fedora)
+		goval-dictionary fetch fedora ${@} 33 34 35
```

Comment:
```suggestion
		goval-dictionary fetch fedora ${@} 32 33 34 35
```

Reply:
then why  `redhat 5` and `debian 7 `,  `ubuntu 14` is not written?

Reply:
OS that is EOL is omitted. So Fedora 32 and 33 is prefer to omit.
https://endoflife.date/fedora

Reply:
I now understand the policy of removing EOL from fetch.